### DISCUSSION TITLE
Gerätenamen ändern

### DISCUSSION THREAD

**Discussion by mario81k** [Blackymas/NSPanel_HA_Blueprint#422] [Q&A] [answered]
Hallo,
ich habe mich dumm angestellt und mein Panel "ns-panel" genannt. Da es durch das "-" zu Problemen kommt würde ich den Namen nun gerne ändern. Kann mir einer helfen wie?
Danke!
Gruß Mario

**Comment by Blackymas** [comment, marked as answer]
Hallo,

einfach den namen im esphome ändern und neu flashen
